README.md

```diff
@@ -3,132 +3,339 @@
+3. If you are not already using the `conda` package manager, install the `conda` mambaforge distribution following https://github.com/robotology/robotology-superbuild/blob/master/doc/install-mambaforge.md#linux. Remember to restart your shell session or run `source ~/.bashrc` (`~/.bash_profile` on MacOS) for the `conda init` command to take effect.
+    Create a new environment and activate it:
     ```
     conda create -n robotologyenv
     conda activate robotologyenv
-    mamba install -c robotology yarp
     ```
-    To read more about installing `robotology` package binaries refer to https://github.com/robotology/robotology-superbuild/blob/master/doc/conda-forge.md#binary-installation.
-2. Install **node.js** and **npm**: we recommend to install them via [NVM](https://github.com/nvm-sh/nvm) (Node Version Manager). Currently, the latest **node.js** LTS version compatible with **YarpJS** `master` commit [be28630](https://github.com/robotology/yarp.js/commit/be2863022713ded2fa48909404b43e98b09eeda2) is the LTS: Argon release **node.js v4.2.2** (refer to https://github.com/robotology/yarp.js/issues/19). Meanwhile, the latest **node.js** LTS version compatible with **Open MCT** is latest LTS:Fermium version **node.js v14.17.0**. For this reason we install both releases.
+    To read more about installing robotology package binaries refer to https://github.com/robotology/robotology-superbuild/blob/master/doc/conda-forge.md#binary-installation.
+4. Install the YARP, icub-models, GazeboYARPPlugins modules
+    ```
+    mamba install -c conda-forge -c robotology yarp gazebo-yarp-plugins icub-models
+    ```
+
+The server installation described in the following section can be done in any machine residing in the same local network as the machine running the simulation. This is relevant in case you wish to run the simulation in some dedicated powerful server machine, and run the visualizer server in your own machine with lower CPU and memory profile. 
+
+## [Server Installation](#top)
+
+### Install NVM, Node.js and npm
+
+⚠️ Run the installation steps 1 and 2 from a new terminal without any `conda` package manager environment enabled.
+
+1. Install [**NVM**](https://github.com/nvm-sh/nvm) (mandatory), a **Node.js** Version Manager by MIT, which safely handles the installation of multiple versions of Node.js and easy switching between versions. As mentioned in later steps, two different versions of Node.js versions are required for installing and running the visualization tool.
+    Follow the NVM installation instruction steps in https://github.com/nvm-sh/nvm, recapped below:
+    
+    a. Run the command below which downloads and runs the one-line installer
+    ```
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
```

Comment:
Note, unless we are specifically requiring for the ``v0.38.0`` version, I would avoid keeping this link here. Is there any link with ``latest`` for example?

Reply:
The problem with using "latest" is that it can select a version we never tested, which is even more likely to happen in the case of a third party tool.

(It actually happened to me when installing the `yarp.js` modules. The IIT maintainers assumed it worked with the "latest" Node.js version, didn't maintain the framework against the later releases, and currently that framework is far behind the latest release, now incompatible.)

Reply:
I can mention that v0.38 is the latest version tested in this context, but the user is free to use his own version if he/she is already familiar with and is using NVM in his workflow.

Reply:
Good point, I agree!